modules/login/php/requestaccount.class.inc

```diff
@@ -32,7 +32,13 @@ class RequestAccount extends \NDB_Form
      */
     function setup()
     {
-        $this->tpl_data['page_title'] = 'Request Account';
+        $this->tpl_data['page_title']  = 'Request Account';
+        $this->tpl_data['captcha_key'] = \NDB_Factory::singleton()
+            ->config()
+            ->getSetting('reCAPTCHAPublic');
+        $this->tpl_data['nonce']       = base64_encode(
+            openssl_random_pseudo_bytes(16)
```

Comment:
I don't see why this is using an openssl extention instead of just using `random_bytes`